builtin/mainmenu/common.lua

```diff
@@ -300,3 +300,34 @@ function is_server_protocol_compat_or_error(proto_min, proto_max)
 
 	return true
 end
+
+--------------------------------------------------------------------------------
+local function menu_load_worldmt(selected)
```

Comment:
I close this PR and let you making the patch, since you seem more competent for it :)